internal/plugin.go

```diff
@@ -342,6 +375,45 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 				policy.Manifests[i].ComplianceType = policy.ComplianceType
 			}
 		}
+
+		// Merge policySets defined in p.policies and p.policyDefaults with p.PolicySets
+		for _, plcsetInPlc := range policy.PolicySets {
```

Comment:
I don't find using additional map too useful here. User can include policy in policyset by either policies[*].policySet or policysets.policies or both. In order to fully populate the two maps, I need to union the results. It would still require to use nested loop to achieve it. Any suggestions or did I miss anything here?